server/api/project_apis.py

```diff
@@ -50,10 +49,15 @@ def get(self, project_id):
             error_msg = f'Project GET - unhandled error: {str(e)}'
             current_app.logger.critical(error_msg)
             return {"error": error_msg}, 500
+        finally:
+            try:
```

Comment:
Add a single line comment saying this call will unlock any tasks older than 2 hours